libyul/YulStack.h

```diff
@@ -133,9 +133,16 @@ class YulStack: public langutil::CharStreamProvider
 		langutil::CharStreamProvider const* _soliditySourceProvider = nullptr
 	) const;
 	Json::Value astJson() const;
+
+	/// Return the EVM assembly in Json format.
+	Json::Value assemblyJson() const;
+
 	/// Return the parsed and analyzed object.
 	std::shared_ptr<Object> parserResult() const;
 
+	/// Collect all source indices of the given Yul object.
```

Comment:
This kind of docstring is pretty useless. It does not say anything you can't already figure out from the name. It should be more specific to be useful:
```suggestion
	/// Collect names of all Solidity source units present in the debug data
	/// of the Yul object (including sub-objects) and their assigned indices.
```